src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,21 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany
```

Comment:
I think we can use ManyToMany Strategy,because the same smiles can be used in different comments
```suggestion
    @ManyToMany(fetch = FetchType.EAGER)
```

Reply:
> I think we can use ManyToMany Strategy,because the same smiles can be used in different comments

\+